docs/safe-haven-services/s3-service.md

```diff
@@ -0,0 +1,145 @@
+# S3 Service
+
+There is no general-purpose S3 service within Safe Haven Services, unlike the [EIDF S3 service](../services/s3/).
+
+However there is a S3 service in SHS with the following caveats:
+* it is only available to the Scottish National Safe Haven, other tenants by arrangement
+* it is a read-only service, as a way of providing access to large collections of files
+* it is not a storage solution for users wanting to create their own files
+
+## Access arrangements
+
+Access to buckets is via keys (Access Key and Secret Access Key) provided to the user by the Research Coordinator.
```

Comment:
Suggested rewording: Some Safe Havens may provide you with access to data via S3. If this applies to your project, your Research Coordinator will provide you with an access key. This documentation will guide you through how to get access to your data from a terminal as well as programmatically via R and Python.

!!! important Files in S3 buckets are read-only. If you need to transform or make changes to any files, you will need to download them to your project space. If you download files, please be mindful of disk space by only downloading what is necessary and deleting them as soon as no longer needed.